### ISSUE TITLE
Last PR broke loading of filters from disk.

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#145]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/145) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

```python
chan 1 skipping compute_filter because already done, and loading filter
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-de6809dcc9de> in <module>()
----> 1 data=ltd2017.load()

/Users/fowlerj/Microcal/Nonlinearity/APrioriCal/sept2017/ltd2017_paper_Ejoule.pyc in load(forceNew)
     22     data.avg_pulses_auto_masks(forceNew=forceNew)
     23     data.compute_noise_spectra(forceNew=forceNew)
---> 24     data.compute_filters(f_3db=10000, forceNew=forceNew)
     25     data.filter_data(forceNew=forceNew)
     26     data.drift_correct(forceNew=forceNew)

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/mass-0.6.2-py2.7-macosx-10.12-x86_64.egg/mass/core/utilities.pyc in work(self, *args, **kwargs)
    110                 print_updater = NullUpdater()
    111 
--> 112             for d in func(self, *args, **kwargs):
    113                 print_updater.update(d)
    114 

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/mass-0.6.2-py2.7-macosx-10.12-x86_64.egg/mass/core/channel_group.pyc in compute_filters(self, fmax, f_3db, forceNew)
    955                 ds.filter = Filter(ds.average_pulse[...], self.nPresamples - ds.pretrigger_ignore_samples,
    956                                    ds.noise_psd[...], ds.noise_autocorr[...], sample_time=self.timebase,
--> 957                                    fmax=fmax, f_3db=f_3db, shorten=2)
    958                 ds.filter.peak_signal = h5grp.attrs['peak']
    959                 ds.filter.shorten = h5grp.attrs['shorten']

TypeError: __init__() got an unexpected keyword argument 'fmax'
> /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/mass-0.6.2-py2.7-macosx-10.12-x86_64.egg/mass/core/channel_group.py(957)compute_filters()
    955                 ds.filter = Filter(ds.average_pulse[...], self.nPresamples - ds.pretrigger_ignore_samples,
    956                                    ds.noise_psd[...], ds.noise_autocorr[...], sample_time=self.timebase,
--> 957                                    fmax=fmax, f_3db=f_3db, shorten=2)
    958                 ds.filter.peak_signal = h5grp.attrs['peak']
    959                 ds.filter.shorten = h5grp.attrs['shorten']
```

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fixes #145 (changed API to create filter).

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
